mobile/Ihc/components/UpdateMedicationModal.js

```diff
@@ -19,41 +19,70 @@ export default class UpdateMedicationModal extends Component<{}> {
    *    showModal: boolean
    *    closeModal: function
    *    saveModal: function
-   *    updateMedication: function
-   *    medicationToEdit: Medication object
+   *    formValues: formValues
    *  }
    */
   constructor(props) {
     super(props);
   }
+  
+  /*MedicationInventoryForm = t.struct({
+    drugName: t.String, // drug name
+    quantity: t.int,
+    dosage: t.int,
+    units: t.String,
+    comments: t.maybe(t.String)
+  });*/
 
-  updateName(drugName) {
-    this.props.medicationToEdit.drugName = drugName;
-    this.props.updateMedication(this.medication);
-  }
+  Units = t.enums({
```

Comment:
Did you check with IHC on what units they want? I don't think kg is a common unit for drugs, whereas mg would be

Reply:
I think those are just dummy measurements for now.. We can easily change it later on

Reply:
Sure if thats the case, can we leave a TODO as a reminder?